driver/main_args.ml

```diff
@@ -731,15 +731,32 @@ let mk_extension f =
     Clflags.Extension.(List.map to_string all)
   in
   "-extension", Arg.Symbol (available_extensions, f),
-  "<extension>  Enable the extension (may be specified more than once)"
+  "<extension>  Enable the specified extension (may be specified more than once)"
+;;
+
+let mk_no_extension f =
+  let available_extensions =
+    Clflags.Extension.(List.map to_string all)
+  in
+  "-no-extension", Arg.Symbol (available_extensions, f),
+  "<extension>  Disable the specified extension (may be specified more than once)"
 ;;
 
 let mk_disable_all_extensions f =
   "-disable-all-extensions", Arg.Unit f,
-  " Disable all extensions, wherever they are specified; this flag\n\
-  \    overrides the -extension flag (whether specified before or after this\n\
-  \    flag), disables any extensions that are enabled by default, and\n\
-  \    ignores any extensions requested in OCAMLPARAM."
+  " Disable all extensions, wherever they have been specified; this\n\
+  \    flag overrides prior uses of the -extension flag, disables any\n\
+  \    extensions that are enabled by default, and causes future uses of\n\
+  \    the -extension or -no-extension flag to raise."
+;;
+
+let mk_only_erasable_extensions f =
+  "-only-erasable-extensions", Arg.Unit f,
+  " Disable all extensions that cannot be \"erased\" to attributes,\n\
```

Comment:
I think this should just list the extensions in play.

Reply:
I changed this to

```
let mk_only_erasable_extensions f =
  let erasable_extensions =
    let open Clflags.Extension in
    all |> List.filter is_erasable |> List.map to_string |> String.concat ", "
  in
"-only-erasable-extensions", Arg.Unit f,
  " Disable all extensions that cannot be \"erased\" to attributes,\n\
  \    wherever they have been specified; this flag overrides prior\n\
  \    contradictory uses of the -extension or -no-extension flags,\n\
  \    raises on future uses of them, and disables any such extensions\n\
  \    that are enabled by default.\n\
  \    (Erasable extensions: " ^ erasable_extensions ^ ")"
;;
```

Is that what you meant?